synopsys-task/src/synopsys-task/utility.ts

```diff
@@ -198,3 +228,31 @@ export function extractBranchName(branchName: string): string {
 
   return branchName.substring(prefix.length);
 }
+
+export function getStatusCode(str: string) {
+  const words = str.split(" ");
```

Comment:
We can put a comment here stating, how we are extracting info from exact locations. May be we can highlight the format of str as a comment

Reply:
@jahid1209 please address this, since these are the changes for return_status.

Reply:
@kishorikumar Added comment.